cmd/sops/subcommand/updatekeys/updatekeys.go

```diff
@@ -45,12 +46,23 @@ func updateFile(opts Opts) error {
 	if err != nil {
 		return err
 	}
-	store := common.DefaultStoreForPath(sc, opts.InputPath)
```

Comment:
I think this should simply be changed to `store := common.DefaultStoreForPathOrFormat(sc, opts.InputPath, opts.InputType)`. That function takes care of everything.